client/.storybook/preview.js

```diff
@@ -26,3 +28,7 @@ export const parameters = {
 };
 
 addDecorator((storyFn, context) => withConsole()(storyFn)(context));
+
+addDecorator((story) => (
+  <MemoryRouter initialEntries={["/"]}>{story()}</MemoryRouter>
```

Comment:
Why did we need this?

Reply:
PinnedContent uses `withRouter` which throws an error if there is no `Router` used